<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StokHareketleriDialog</class>
 <widget class="QDialog" name="StokHareketleriDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Stok Hareketleri</string>
  </property>
  <widget class="QTableView" name="stokhareketleritableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>51</y>
     <width>981</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>560</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Başlangıç tarihi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="baslangicdateEdit">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Bitiş  tarihi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="bitisdateEdit">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
